package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "vali-ah-frontend",
+  "version": "1.0.0",
+  "description": "The one-stop shop for all the self-help ideas you need.",
+  "main": "index.jsx",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node ./serve/app ",
```

Comment:
I wonder if this line was tested since it does not run on my system.
<img width="540" alt="Screenshot 2019-08-27 at 8 17 34 AM" src="https://user-images.githubusercontent.com/22734663/63749408-4a1ab700-c8a3-11e9-9ac2-4240410c1a5d.png">

Reply:
Thanks @iverenshaguy 
You have to run build first before you run start.
Also, that is for production which works on heroku.
For development, just run 'npm run start:dev'

Reply:
Really, so production cannot work on my system? Also, everybody knows you have to run build before starting in production. This command does not work. Please fix it.